packages/explat-client/README.md

```diff
@@ -1,8 +1,75 @@
+# ExPlat Client
 
-**NOT READY FOR USE**
+This is a standalone client for Automattic's ExPlat, allowing use of ExPlat in any Javascript context.
 
-# ExPlat Client
+## Self-Contained
+
+`const exPlatClient = createExPlatClient(config)`
+
+- Dep injects outside parts so it can be fitted to any codebase.
+- Doesn't assume much of its environment.
+- No external dependencies or libs need.
+- Stores state internally and simply.
+
+## Type: `ExperimentAssignment`
+
+`ExperimentAssignment` represents an experiment assignment, as an experimenter you just need to look at `ExperimentAssignment.variationName`.
+
+- `variationName === null`: This means you should return the default experience.
+- `variationName !== null`: This means you should return the treatment experience. Currently `variationName` will always be `treatment`, but this may change.
+
+This type will likely be extended, it can also be missing in some API functions, particularly in the React side of things. A missing experiment assignment _does not_ mean the default experience, it means we do not have an assignment yet and if we can afford to wait we should, generally displaying a loading experience.
```

Comment:
Would it be worth clarifying that the client doesn't have an assignment, but the user **may** have an assignment?

Reply:
Not too sure what the difference here is?

Reply:
There have been cases in the past where people want to have A/B/C experiments where people not eligible for the the experiment get a different experience. As it's written now, people may be misled to think that the user doesn't have a variation (which may or may not be true) and then show a different experience assuming that means the user isn't eligible for the experiment.

I don't think it's a big issue, so I'm fine with keeping it; just thought I'd point that case out.

Reply:
Actually, reading the whole thing again. I don't think it could be misleading.